locales/ja.json

```diff
@@ -0,0 +1,289 @@
+{
+  "components": {
+    "airport_select": {
+      "placeholder": "空港の選考"
+    },
+    "color_select": {
+      "placeholder": "色彩の選考"
+    },
+    "line_index_select": {
+      "custom_indices": "路線記号の選考",
+      "line_name": "{index}線",
+      "official_indices": "公式の路線記号",
+      "placeholder": "特注の路線記号"
+    },
+    "map_size_select": {
+      "placeholder": "サイズの選考"
+    },
+    "mode_select": {
+      "placeholder": "交通機関の選考"
+    },
+    "service_select": {
+      "placeholder": "交通サービスの選考"
+    },
+    "shape_select": {
+      "placeholder": "形の選考"
+    }
+  },
+  "data": {
+    "airports": {
+      "both": "CDG + ORY",
+      "cdg": "パリ＝シャルル・ド・ゴール空港",
+      "generic": "空港",
+      "ory": "パリ＝オルリー空港"
+    },
+    "colors": {
+      "custom": "特注の色"
+    },
+    "map_size": {
+      "big": "XL (30em)",
+      "huge": "XXL (50em)",
+      "large": "L (20em)",
+      "normal": "M (15em)"
+    },
+    "mode": {
+      "bike": "自転車",
+      "boat": "水上バス",
+      "brt": "高速バス",
+      "bus": "バス",
+      "cable": "ロープウェイ",
+      "metro": "地下鉄",
+      "rer": "RER",
+      "tram": "路面電車",
+      "transilien": "Transilien"
+    },
+    "services": {
+      "airport": "空港",
+      "bullet_train": "新幹線",
+      "cdgval": "CDGVAL",
+      "funicular": "ケーブルカー",
+      "long_distance_bus": "高速バス",
+      "main_station": "主要駅",
+      "orlyval": "OrlyVal",
+      "suburban_train": "通勤列車",
+      "ter": "TER",
+      "tgv": "TGV"
+    },
+    "shapes": {
+      "circle": "丸",
+      "lines": "線",
+      "rectangle": "長方形",
+      "rounded_square": "角丸四角形"
+    }
+  },
+  "locales": {
+    "en": "英語",
+    "fr": "フランス語",
+    "ja": "日本語"
+  },
+  "misc": {
+    "beta": "ベータ版",
+    "temporary": "臨時"
+  },
+  "ui": {
+    "dialogs": {
+      "browser_check": {
+        "dismiss": "分かりました",
+        "header": "ブラウザはサポートされていません",
+        "summary_1": "ブラウザとして<b>{browserName}</b>を使用しているようです。このブラウザはBULBでは正式にサポートされていません。このまま使用を続けると、予期せぬ動作が発生したり、グラフィックの問題が発生する可能性があります。",
+        "summary_2": "最適な体験のためには、Google Chrome、Microsoft Edge、Opera、BraveなどのChromiumベースのブラウザを使用することをお勧めします。",
+        "supported_browsers": "サポートされているブラウザ",
+        "unsupported_browsers": "サポートされていないブラウザ"
+      },
+      "connections_editor": {
+        "group": {
+          "delete": "消す",
+          "header": "{index}番目の乗り換えのグループ",
+          "mode": {
+            "lines": {
+              "add": "新しい線路",
+              "header": "線路"
+            },
+            "mode": "交通機関"
+          },
+          "ornament": "装飾品",
+          "pedestrian_connection": "歩きで乗り換える",
+          "service": {
+            "services": {
+              "add": "新しいサービス",
+              "header": "交通サービス"
+            }
+          }
+        },
+        "header": "{stopName}駅の乗り換え",
```

Comment:
Missing quotes around the stop name

Reply:
There are no quotes in Japanese for this. It would be strange to use them I think. The 駅 is used as a suffix to specify it's a station.
If you absolutely want to use quotation marks, you can use: `「{stopName}」駅の乗り換え`

Reply:
I see what you mean. Considering a Japanese user would name their stops in Japanese, it is fine as it. I just tough of the case of a Latin name in a Japanese UI, but this is secondary.

Reply:
even then, the quotes are redundant. in Japanese, the only real use case for quotes is literally to quote someone